### PR TITLE
Creating data model

### PR DESCRIPTION
**Pull request by AlaaAlsalem**
Here are the things I did to this repo
> Create the project with the PostgreSQL database
>Buld the project's schema

### CODE REVIEW

File: db/migrate/20220827115952_create_users.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :string
```
Comment (ahmetbozaci) [inline db/migrate/20220827115952_create_users.rb]: - [ ] There is no `string` column in the schema. Kindly delete it. User table only name, photo, bio and post counter column.

File: db/migrate/20220827125506_add_user_ref_to_comments.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddUserRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
+  end
```
Comment (ahmetbozaci) [inline db/migrate/20220827125506_add_user_ref_to_comments.rb]: - [ ] Reference name should be an author. Kindly fix it. [Requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png)

File: db/migrate/20220827125322_add_user_ref_to_posts.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class AddUserRefToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
```
Comment (ahmetbozaci) [inline db/migrate/20220827125322_add_user_ref_to_posts.rb]: - [ ] Reference name should be an author. Kindly fix it. [Requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png)

File: db/migrate/20220827125808_add_user_ref_to_likes.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+class AddUserRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
```
Comment (ahmetbozaci) [inline db/migrate/20220827125808_add_user_ref_to_likes.rb]: - [ ] Reference name should be an author. Kindly fix it. [Requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png)

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️


 @AlaaAlsalem  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Highlights 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
almost

___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

File: db/migrate/20220827125506_add_user_ref_to_comments.rb (hunk 2 above)
Comment (AlaaAlsalem) [reply db/migrate/20220827125506_add_user_ref_to_comments.rb]: thanks for your feedback, I fixed them all

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**